user-guide/manual.adoc

```diff
@@ -344,24 +344,38 @@ Adding an Agent to the ANMS can be done from the Agents page as well. Enter the
 .Add an Agent to the ANMS
 image::images/agent-add.png[]
 
-To manage an Agent, enter the address(es) of the Agent(s), using a comma separated list of addresses if multiple Agents are to be managed at once. 
-Click the `Manage agent` button when this field has been populated to pull up a menu displaying Agent management options. 
+To manage an Agent(s), select the agent(s) to be managed from the agent table, then select the manage button in the upper right part of the table. 
 
 [#fig-agent-manage]
 .Manage an Agent 
 image::images/agent-manage.png[]
 
 From this Agent management menu, the user can:
-1. De-register the Agent.
-2. Send a Time Based Rule (TBR). 
-3. Send a raw command (in hex).
-4. Print the reports generated by the Agent.
-5. Write the reports generated by the Agent to a file (feature not currently supported).
+1. Generate a new ARI to send to the agent(s)
+3. Send a prebuilt string ARI to the agent(s)
```

Comment:
Seems like a number is missed. To avoid misnumbering you can use all "1. " prefix and the renderer will auto number the list.

Reply:
updated to us 1. for all entries